web/src/components/Contact.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+import { useTheme } from 'styled-components'
+
+import ContactModel from 'shared/api/models/ContactModel'
+
+import { MailIcon, PhoneIcon, WebsiteIcon } from '../assets'
+import Collapsible from './Collapsible'
+import ContactItem from './ContactItem'
+import Spacer from './Spacer'
+
+const Contact = ({ contact: { headline, website, phoneNumber, email } }: { contact: ContactModel }): ReactElement => {
+  const theme = useTheme()
+  const { t } = useTranslation('pois')
+
+  return (
+    <>
+      <Spacer $borderColor={theme.colors.borderColor} />
```

Comment:
:wrench: I think this spacer should not be part of this component to keep it reusable in different places.
``` suggestion
```